features/financial_assistance/step_definitions/other_questions_page_steps.rb

```diff
@@ -253,6 +253,10 @@
   choose('is_pregnant_no')
 end
 
+And(/the post partum language one year is not enabled/) do
```

Comment:
We have a helper set up that makes defining this step unnecessary.  We should be able to just add: 
```
And FAA post_partum_period_one_year feature is enabled
```
to the other_questions_pregnancy.feature file.  

Check out features/support/worlds/resource_registry_world.rb for the implementation details.